yt/yt/server/lib/misc/ya.make

```diff
@@ -31,6 +31,8 @@ PEERDIR(
     yt/yt/library/tcmalloc
     yt/yt/library/ytprof
     yt/yt/library/ytprof/allocation_tag_profiler
+    yt/yt/library/containers/cri
```

Comment:
Why are these dependencies needed in lib/misc if there are no other changed files?